code/modules/mob/new_player/sprite_accessories_ear.dm

```diff
@@ -517,4 +517,14 @@
 	icon_state = "elfs"
 	do_colouration = 1
 	color_blend_mode = ICON_MULTIPLY
-	extra_overlay = "dragon-horns"
\ No newline at end of file
+	extra_overlay = "dragon-horns"
+
+/datum/sprite_accessory/ears/zorgoia
+	name = "Zorgoia"
```

Comment:
call it something more like zorgoia ears pls? Mostly for clarity

Reply:
added ears. Like the text, the ears were already there. But now they're "Zorgoia ears"